.env_example

```diff
@@ -15,10 +18,20 @@ APRUVE_CORPORATE_ACCOUNT_ID="your-corporate-account-id"
 # local - localhost
 #
 APRUVE_ENVIRONMENT="test"
+export APRUVE_ENVIRONMENT
 
 # URL of the Apruve credit app
 APRUVE_CREDIT_APP_URL="http://localhost:3000/apply/munder-difflin-inc"
+export APRUVE_CREDIT_APP_URL
 
 APRUVE_OAUTH_CLIENT_ID="oauth-client-id"
+export APRUVE_OAUTH_CLIENT_ID
 APRUVE_OAUTH_CLIENT_SECRET="oauth-client-secret"
-APRUVE_OAUTH_REDIRECT_URL="an-ssl-address"
\ No newline at end of file
+export APRUVE_OAUTH_CLIENT_SECRET
+APRUVE_OAUTH_REDIRECT_URL="an-ssl-address"
+export APRUVE_OAUTH_REDIRECT_URL
```

Comment:
We don't need any of the exports here since the .env file isn't ever really "run" it is just used for config variables. If we keep the `export` statements in the README then people will know to call them.

Reply:
Also you can just add to the README manually through github

Reply:
I've manually edited README and created a new pull request on a new branch

Reply:
Sweet thank you!